### PR TITLE
Add Peekaboo view as a subview to key Window so that it is still visible if root VC is changed

### PR DESCRIPTION
**Pull request by ab1d-rahman**

### CODE REVIEW

**Comment by ratulSharker** [comment]
Hi @ab1d-rahman ,

I have added a `rootViewController` example so that we can check changing `rootViewController` still shows the view. Your implementation is working fine. Another thing we needed to track the view which was passed via:
```
public func peek(view: UIView, height: CGFloat)
```

Otherwise, while showing different `UIView` by passing it to `peek` function, previous view will be still attached to `UIWindow`.

For example say we show `view1`, then hide it. After hiding we show `view2`. In this scenario `view1` is still be attached to `UIWindow`.

To remedy this situation, we can track the view attached by introducing a nullable private property. Assign to this view property while `peek` function is called. While calling hide, call `view?.removeFromSuperView()` so that it hides while hide animation completes.

File: RSPeekaboo/Classes/RSPeekaboo.swift [hunk 1]
```diff
@@ -8,95 +8,77 @@ import UIKit
...
-        UIView.animate(withDuration: 0.5, animations: { // Allow customizing times
-            rootView.frame = finalRootViewFrame
-            view.frame = finalViewFrame
-            rootView.layoutIfNeeded()
-        }) { (finished) in
-            // Use weak reference
-            self.mPeekabooState = .shown
+
+        self.mPeekabooState = .animating
+        self.mLastShownView = view
```
Comment (ab1d-rahman) [inline RSPeekaboo/Classes/RSPeekaboo.swift:42]: Hi @ratulSharker vai, referring to your comment [here](https://github.com/ratulSharker/RSPeekaboo/pull/7#issuecomment-610254346),

I have kept a reference of the passed view in `peek` method here. This is like your previous implementation. I have not altered anything here. This `mLastShownView` is then used in `hide` method to remove this view from the window.

**Comment by ab1d-rahman** [comment]
Another thing, as I have used autolayout to put constraints on the `view` that is passed to the `peek` method so that it is positioned in the window correctly, the passed `view` must have its `translatesAutoresizingMaskIntoConstraints` flag set to `false` for this to work.

File: RSPeekaboo/Classes/RSPeekaboo.swift (hunk 1 above)
Comment (ratulSharker) [reply RSPeekaboo/Classes/RSPeekaboo.swift:42]: Don't know why i skipped this. My bad 
Btw good work (Y)

**Comment by ratulSharker** [comment]
> Another thing, as I have used autolayout to put constraints on the `view` that is passed to the `peek` method so that it is positioned in the window correctly, the passed `view` must have its `translatesAutoresizingMaskIntoConstraints` flag set to `false` for this to work.

So we are open to two option here

1. Set the `transaltesAutoresizingMaskIntoConstraints` within the `peek` to `false`.
2. Or guide pod users to do so.

Second option is always available for us, as there is nothing to do :P . But for the first option we need to do some testing. Example cases like an `UIView` having several subviews, and subview's having more subviews which are layout using frame base layout, how it behaves while setting `transaltesAutoresizingMaskIntoConstraints`  to `false`. Keep it as another issue.

**Pull request merged**